Functions.Templates/Templates/SqlInputBinding-JavaScript/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = async function (context, results) {
+    if (!!results && results.length > 0) {
+        context.log('Query results: ', JSON.stringify(results));
```

Comment:
We should do something besides just log - make sure you're returning the correct object so that the trigger shows the results as expected.

Reply:
I return the results in our samples, but again following the cosmosdb sample so didn't.

Reply:
Let's do what we think makes sense and not just copy them - given that our samples are pretty much all returning the items retrieved let's do that here too.